src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+@import "./utils/variables";
+@import "./utils/function";
+@import "./utils/mixin";
+
+.form {
```

Comment:
You dont need to use this block, as i wrote before, otherwise you should move this block to the separate file

Reply:
I use this block form for center position for my calendar